help_to_heat/frontdoor/views.py

```diff
@@ -358,8 +360,50 @@ def save_data(self, request, session_id, page_name, *args, **kwargs):
 @register_page("address")
 class AddressView(PageView):
     def handle_post(self, request, session_id, page_name, data, is_change_page):
-        return redirect("frontdoor:page", session_id=session_id, page_name="address-select")
+        data = interface.api.session.get_answer(session_id, "address")
+        building_name_or_number = data["building_name_or_number"]
+        postcode = data["postcode"]
+        try:
+            interface.EPC.get_address_and_epc_rrn(building_name_or_number, postcode)
```

Comment:
Could add a condition here to only make this call if the country isn't Scotland to decrease the number of API calls we make

Reply:
Probably worthwhile, yeah.